backend/src/media/backends/imgur-backend.ts

```diff
@@ -11,15 +11,19 @@ import mediaConfiguration, { MediaConfig } from '../../config/media.config';
 import { MediaBackendError } from '../../errors/errors';
 import { ConsoleLoggerService } from '../../logger/console-logger.service';
 import { MediaBackend } from '../media-backend.interface';
-import { BackendData } from '../media-upload.entity';
 
 type UploadResult = {
   data: {
     link: string;
-    deletehash: string;
+    deletehash: string | null;
   };
 };
 
+interface BackendData {
```

Comment:
This should probably be `ImgurBackendData` if only to mitigate confusion to the old `BackendData`